src/main/kotlin/nl/info/zac/productaanvraag/ProductaanvraagService.kt

```diff
@@ -411,44 +419,51 @@ class ProductaanvraagService @Inject constructor(
         )
     }
 
-    private fun assignZaakToGroup(zaak: Zaak, groupName: String) {
-        LOG.info("Assigning zaak with UUID '${zaak.uuid}' to group: '$groupName'")
-        zrcClientService.createRol(creeerRolGroep(groupName, zaak))
+    private fun assignZaakToGroup(zaak: Zaak, groupName: String, productaanvraagDimpact: ProductaanvraagDimpact) {
+        createRolGroep(groupName, zaak)?.let {
+            LOG.info("Assigning zaak with UUID '${zaak.uuid}' to group: '$groupName'")
+            zrcClientService.createRol(it)
+        } ?: LOG.warning(
+            "Missing behandelaar roltype for zaaktype UUID '${zaak.zaaktype.extractUuid()}'. " +
```

Comment:
Missing behandelaar roltype? Why would that be? Is this correct? Seems like strange logic to me.